internal/peer/chaincode/common_test.go

```diff
@@ -184,8 +188,11 @@ func TestGetOrdererEndpointFail(t *testing.T) {
 	assert.NoError(t, err)
 
 	mockchain := "mockchain"
-	factory.InitFactories(nil)
-
+	// Init the BCCSP
+	err = factory.InitFactories(nil)
+	if err != nil {
+		assert.Error(t, err, "Could not initialize BCCSP Factories")
```

Comment:
Error is not expected here, so why assert it?
BTW expected error would go before actual error in assert.Error().

Reply:
Use assert.NoError(t, err) which will flag unexpected introduced behavior